src/serialmonitor/serialportctrl.ts

```diff
@@ -31,134 +28,4 @@ export class SerialPortCtrl {
       };
```

Comment:
Can the Serial Monitor extension provide port enumeration instead? It's a shame that we still need a dependency on `serialport` for this one call.